include/gatekeeper_gk.h

```diff
@@ -88,6 +88,14 @@ struct gk_config {
 	unsigned int       gk_max_num_ipv4_fib_entries;
 	unsigned int       gk_max_num_ipv6_fib_entries;
 
+	/*
+	 * Timeout in seconds used to prune the expired request flow entries.
+	 */
+	unsigned int       request_timeout_sec;
+
+	/* Timeout for scanning the whole flow table in ms. */
+	unsigned int       flow_table_scan_timeout_ms;
```

Comment:
1. Replace `Timeout` in the comment to `Time.

2. Rename `flow_table_scan_timeout_ms` to `flow_table_full_scan_ms`.